CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Allowed Types of change: `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `
 ## Unreleased
 
 - SC-9231 - add permissions to SuperHero to edit admin role 
+- SC-9130 - use whiltelisted filenames at the s3 file storage backend
```

Comment:
Please look into the util generateFlatFileName it is more s3 related. But we have also sanitation that should prevent us for some changes if name is one of the keys that are filtered. But in generally i think only the the service / module should now how it sanitize his stuff.

Reply:
not found that

Reply:
now, one mane is used only